users/models/identity.py

```diff
@@ -812,6 +813,16 @@ def to_opengraph_dict(self) -> dict:
 
     ### Mastodon Client API ###
 
+    @cache_memoize(120)
+    def mention_to_mastodon_json(self):
+        return {
+            "id": self.id,
+            "username": self.username or "",
+            "url": self.absolute_profile_uri() or "",
+            "acct": self.handle or "",
+        }
+
+    @cache_memoize(120)
```

Comment:
This is only used by Identity search from the clients. How likely are users on the same instance to search for the same Identity within a 2 minute window. I think this may be an incurred cost without any realized benefit.

Reply:
Will be removed